src/sidebar/components/menu-keyboard-navigation.js

```diff
@@ -0,0 +1,99 @@
+import { createElement } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
+import propTypes from 'prop-types';
+
+function isElementVisible(element) {
+  return element.offsetParent !== null;
+}
+
+/**
+ * Menu wrapper component to facilitate keyboard navigation of a list of
+ * <MenuItem> components.
```

Comment:
I think it would be useful to clarify that this is a helper component used by `Menu` and `MenuItem` to wrap a list of `MenuItem`s, and that other components shouldn't need to use it directly.